nginx.conf

```diff
@@ -0,0 +1,30 @@
+user nginx;
+worker_processes auto;
+worker_cpu_affinity auto;
+
+events {
+  worker_connections 1024;
+}
+
+# Process needs to run in foreground within container
+daemon off;
+
+http {
+  server {
+    listen 80;
+    root /usr/share/nginx/html;
+    include /etc/nginx/mime.types;
+
+    gzip on;
+    gzip_types text/plain application/javascript application/x-javascript text/javascript text/xml text/css;
+    gzip_min_length 1000;
+
+    location /api {
+        proxy_pass http://ecrnow-ui:8080;
+    }
+
+    location / {
+        proxy_pass http://ecrnow-api:3000;
+    }
```

Comment:
why does `/api` go to `ecrnow-ui` 
and `/` goes to `ecrnow-api`? that looks backwards.

Reply:
oops. Addressing.